src/test/basics/base_uint_test.cpp

```diff
@@ -68,15 +93,15 @@ struct base_uint_test : beast::unit_test::suite
         {
             BEAST_EXPECT(d == --t);
         }
```

Comment:
What do you think about extracting this block of testing into a separate function as it's nearly identical to the block that starts on line 105?